git/00.setup.md

```diff
@@ -42,8 +42,8 @@ combinations.
 ```bash
 git config --global alias.aa add --all
 git config --global alias.d diff --staged
-git config --global alias.count shortlog -sn
-git config --global alias.l log --pretty=format:%C(yellow)%h%Cblue%d%Creset %s %C(white) %an, %ar%Creset --graph
+git config --global alias.pf push --force-with-lease
+git config --global alias.l log --graph --decorate --oneline
```

Comment:
The other format is my personal one I've used for years based on the git-smart gem.  Would need to investigate what is different